### PR TITLE
fix(security): Fix CVE-2017-5929 security issue threat level 9

### PR DESCRIPTION
**Pull request by jingsfdc**
https://nvd.nist.gov/vuln/detail/CVE-2017-5929

QOS.ch Logback before 1.2.0 has a serialization vulnerability affecting the SocketServer and ServerSocketReceiver components. 

The RemoteStreamAppenderClient class in logback-classic and the SocketNode classes in logback-classic and logback-access allow data to be deserialized over a Java Socket, via an ObjectInputStream, without validating the data beforehand. When data is received from the Socket, to be logged, it is deserialized into Java objects. An attacker can exploit this vulnerability by sending malicious, serialized Java objects over the connection to the Socket, which may result in execution of arbitrary code when those objects are deserialized.Note that although logback-core is implicated by the Logback project here, the Sonatype Security Research team discovered that the vulnerability is actually present in the logback-classic and logback-access components.Advisory Deviation Notice: The Sonatype Security Research team discovered that this vulnerability was first introduced in version 0.3.0 of logback-classic and version 0.6.0 of logback-access, as opposed to being present in all versions prior to 1.2.0, as stated in the advisory.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2014 Netflix, Inc.
+ * Copyright (c) 2018, salesforce.com, inc. All rights reserved.
```
Comment (lwander) [inline build.gradle]: Is this a requirement from your legal department? If it's not, the convention is to leave the headers pointing at the original authors.

**Comment by lwander** [comment]
Thanks for this! For future patches like this, can you link to where the CVE was reported (which CVE repo)?

File: build.gradle (hunk 1 above)
Comment (jingsfdc) [reply build.gradle]: @lwander yes, it's a requirement from the legal department.

**Comment by jingsfdc** [comment]
@lwander I have updated the description with the link to the CVE repo. Thanks for the feedback.

**Comment by lwander** [comment]
@spinnakerbot cherry-pick 1.11

File: build.gradle (hunk 1 above)
Comment (jamesward) [reply build.gradle]: Typically a CLA is used to transfer the copyright but it doesn't appear that Spinnaker has a CLA.  @lwander can you confirm?

Comment (lwander) [reply build.gradle]: Yes that's right, there is no CLA at this time

**Pull request merged**